packages/puppeteer-core/Herebyfile.mjs

```diff
@@ -52,17 +51,54 @@ export const generateInjectedTask = task({
   },
 });
 
+export const generatePackageJsonTask = task({
+  name: 'generate:package-json',
+  run: async () => {
+    await mkdir('lib/esm', {recursive: true});
+    await writeFile('lib/esm/package.json', JSON.stringify({type: 'module'}));
+  },
+});
+
 export const generateTask = task({
   name: 'generate',
-  dependencies: [generateVersionTask, generateInjectedTask],
+  dependencies: [
+    generateVersionTask,
+    generateInjectedTask,
+    generatePackageJsonTask,
+  ],
 });
 
 export const buildTscTask = task({
   name: 'build:tsc',
   dependencies: [generateTask],
   run: async () => {
     await execa('tsc', ['-b']);
-    await execa('rollup', ['-c', 'rollup.third_party.config.mjs']);
-    await writeFile('lib/esm/package.json', JSON.stringify({type: 'module'}));
+  },
+});
+
+export const buildTask = task({
+  name: 'build',
+  dependencies: [buildTscTask],
+  run: async () => {
+    const packages = ['mitt', 'rxjs'];
```

Comment:
Can we get the folders and deduce it from there, that ways if we add new one (example `chromium-bidi`) we don't need to add it here?